README.txt

```diff
@@ -3,4 +3,4 @@
     C:\Users\jkotsis\AppData\Roaming\NetBeans\8.2\apache-tomcat-8.0.27.0_base\conf\tomcat-users.xml 
     username="ide" password="tB5GDrpP"
     if you want to logon from the web interface use the user with manager-gui role in the same file
-3. 
\ No newline at end of file
+3. can you tell me the defoult password in order to login in the system plz????????
```

Comment:
It's been a while ago I worked with this. I have to check my work computer. I will be there on Tuesday next week!

Reply:
can u check for me the password hope that you reach at computer work, thank you so much